domains/auth/Auth.API/API/API.csproj

```diff
@@ -9,36 +9,36 @@
     <ItemGroup>
         <PackageReference Include="AspNetCore.HealthChecks.NpgSql" Version="8.0.1" />
         <PackageReference Include="AspNetCore.HealthChecks.Rabbitmq" Version="8.0.1" />
-        <PackageReference Include="EnergyOrigin.TokenValidation" Version="3.1.4" />
-        <PackageReference Include="Grpc.AspNetCore" Version="2.62.0" />
-        <PackageReference Include="EnergyOrigin.IntegrationEvents" Version="1.3.4" />
+        <PackageReference Include="EnergyOrigin.TokenValidation" Version="3.6.8" />
+        <PackageReference Include="Grpc.AspNetCore" Version="2.65.0" />
```

Comment:
Breaking change. We can introduce this. It will make so I cant build code

Reply:
Which one is a breaking change?

Reply:
Der er en bug med den seneste version af grpc.AspNetCore, som gør at folk med ÆØÅ i deres navn ikke kan bygge projekter lokalt. Undlad at opdatere GRPC versionen, så har vi intet problem

Reply:
Alt der hedder noget med GRPC, det er fyfy!